src/commands.cc

```diff
@@ -1045,6 +1046,37 @@ Vector<String> params_to_shell(const ParametersParser& parser)
     return vars;
 }
 
+String deindent_docstring(StringView docstring)
+{
+    if (docstring.empty())
+        return {};
+    else if (docstring[0_byte] != '\n')
+        return trim_whitespaces(docstring).str();
+
+    String ds = docstring.substr(1_byte).str();
```

Comment:
`ds` is not very expressive, I'd rather maybe reuse the parameter (str = str.substr(1_byte)) ? Also, why a String instead of a StringView here ?

Reply:
It's short for "docstring", I didn't really think hard on the variable name.

I created a string because I thought I would modify it directly at first, but then I used `transform` and didn't change the type back.